scripts/ci-reference-docs-metrics.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# abort if we get any error
+set -e
```

Comment:
Adding `o pipefail` should ensure pipelines also fail I think if one of the steps fails.
```suggestion
set -eo pipefail
```